eclair-core/src/test/scala/fr/acinq/eclair/io/SwitchboardSpec.scala

```diff
@@ -20,7 +20,7 @@ class SwitchboardSpec extends TestKitBaseClass with AnyFunSuiteLike {
 
   test("on initialization create peers") {
     val nodeParams = Alice.nodeParams
-    val (probe, peer) = (TestProbe(), TestProbe())
+    val (probe, peer, postman) = (TestProbe(), TestProbe(), TestProbe())
```

Comment:
I think you can remove all the changes made to this file now that the `Switchboard` doesn't depend on the `postman`.